test/babel/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["stage-3"]
```

Comment:
I wonder if using `stage-x` is ideal. I mean, someone know if `stage` presets will evolve in time with ESxx ?
If it's the case, using transformers directly could be better no ?

```js
{
  "plugins": ["transform-async-to-generator"]
}
```